packages/atomic-layout/src/hooks/useCurrentBreakpoints.ts

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react'
+import useViewportChange from './useViewportChange'
+import { Layout, createMediaQuery } from '@atomic-layout/core'
+
+/**
+ * Returns a list of breakpoints that match the current state of the viewport.
+ */
+export default function useCurrentBreakpoints(): string[] {
```

Comment:
Returning a list of breakpoint names, because it's possible for multiple breakpoints to match a single state of the viewport:

```js
Layout.configure({
  breakpoints: {
    portrait: {
      orientation: 'portrait'
    },
    mobile: {
      maxWidth: 576
    }
  }
})
```

On a viewport with the width of `500` both `portrait` and `mobile` breakpoints would match.